lib/utils/language.js

```diff
@@ -14,11 +14,16 @@ define(['polyglot', 'moment', 'helper'], function (Polyglot, moment, helper) {
       }
     }
 
-    function setLocale(event) {
+    function setSelectLocale(event) {
       localStorage.setItem('language', getLocale(event.target.value));
```

Comment:
is nur der event listener und das navigieren passiert sowieso und triggered alles, da URL Änderungen von Hand ja auch gehen sollen